### DISCUSSION TITLE
Resetting Database when changed Enviroment Variables

### DISCUSSION THREAD

**Discussion by PhilippWu** [fastapi/full-stack-fastapi-template#1133] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
-.-
```

### Description

Hey Community,

in my first deployment i setup First Super User with a username not a Email-Adress - so i am not able to login into the deployment.

Is there a way to reset the whole application without reinstalling the server?

### Operating System

Linux

### Operating System Details

Ubuntu with Docker Engine and Github Agent

### Python Version

latest

**Comment by YuriiMotov** [comment, marked as answer]
You can just remove the volume of DB.
* Stop containers using `docker compose down` command.
* Use `docker volume ls` command to get the list of volumes.
* The volume of DB will have the name like `{project-name}_app-db-data` (where `{project-name}` is the name of your project).
* Then use `docker volume rm {project-name}_app-db-data` to remove the volume.
